certified-connectors/Redque/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3548 @@
+{
+    "swagger": "2.0",
+    "basePath": "/",
+    "consumes": [],
+    "host": "demoidentity.redque.com",
+    "info": {
+        "title": "Redque",
+        "description": "Accessing and managing the Redque system",
```

Comment:
@jirisodomka please add a period at the end of the description. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
@Amjed-Ayoub You mean in the sentence? But in the description, you mentioned is sentence without period, right?
![image](https://user-images.githubusercontent.com/118742785/203784915-17f2f926-2dc4-4432-a53a-d0cb6dd7e59a.png)
Or did you mean different description? Anyway, I will update it here.  Thank you.